### DISCUSSION TITLE
HOW TO UPDATE V 1.1.0 TO 2.1.0

### DISCUSSION THREAD

**Discussion by nikedelick** [seaky/nspanel_pro_tools_apk#89] [General]
hi everybody
i m newbies
i need some help to update my ns panel pro V1.1.0 to 2.1.0
Can someone tell me how to do it please?
Thanks a lot

**Comment by seaky** [comment]
Hi,

Do you mean nspanel pro app from 1.1 to 2.1? Or Nspanel pro OS firmware?
